app/helpers/application_helper/button/catalog_item_button.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationHelper::Button::CatalogItemButton < ApplicationHelper::Button::Basic
+  include ApplicationHelper
+
+  def skip?
+    # when user has the privilege to edit Catalog Item, he can also add/edit buttons/button groups
+    !%w(catalogitem_new catalogitem_edit atomic_catalogitem_new atomic_catalogitem_edit).any? do |feature|
```

Comment:
This looks suspicious.

For one, I don't understand why any RBAC features should be tested here. The RBAC is handled elsewhere for buttons.

Second: if we really do need RBAC check the we surely already have a method to test a list of features.